ui/src/app/projects/projects.component.ts

```diff
@@ -79,13 +78,14 @@ export class ProjectsComponent implements OnInit {
     this.projectsObservable = Observable.fromPromise(
       this.projectsService.listProjects(filter)
         .then(listView => {
-          this.projects = listView.results ? listView.results : [];
+          this.projects = listView.results.sort() ? listView.results : [];
```

Comment:
FYI: I can't tell what order CRM is returning these in by default, but it's definitely not ideal since I found the prefix case: `google.com:verily` search does not provide itself as the first result, but instead some other longer project is given.

Reply:
Acknowledged, yeah I couldn't tell either but sorting seems to give us a reasonable order.